spec/spec_helper.rb

```diff
@@ -1,10 +1,23 @@
 # This file is managed via modulesync
 # https://github.com/voxpupuli/modulesync
 # https://github.com/voxpupuli/modulesync_config
+RSpec.configure do |c|
+  c.mock_with :rspec
```

Comment:
Is this why the stub is failing, was this expected to be `:mocha`? I am not well versed in mocks but seem to recall that being the default previously.

Reply:
I did a rewrite of the test #484 based on https://github.com/voxpupuli/puppet-autofs/pull/146/commits/491d6fc5a53d56897fc7bf50f532a2cccbf21f74. Unfortunately I do not have write permissions to this repository, so I was forced to create an additional pull request.